examples/reference/elements/matplotlib/VectorField.ipynb

```diff
@@ -78,9 +79,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts VectorField [magnitude=dim('Magnitude').norm()*0.2 rescale_lengths=False] (color='Angle')\n",
-    "hv.HoloMap({phase: hv.VectorField([x, y,(vector_data[2]+phase)%np.pi*2, vector_data[3]+np.abs(phase)])\n",
-    "            for phase in np.linspace(-np.pi,np.pi,5)}, kdims='Phase')"
+    "hmap = hv.HoloMap({phase: hv.VectorField((x, y,(vector_data[2]+phase)%np.pi*2, vector_data[3]+np.abs(phase)))\n",
+    "                   for phase in np.linspace(-np.pi,np.pi,5)}, kdims='Phase')\n",
+    "\n",
+    "hmap.opts(opts.VectorField(color='Angle', magnitude=dim('Magnitude').norm()*0.2, rescale_lengths=False))"
```

Comment:
I would format this consistently i.e start `opts.VectorField` indented on the next line.